File: app/src/main/java/com/georgcantor/wallpaperapp/ui/fragment/LicenseFragment.java
package com.georgcantor.wallpaperapp.ui.fragment;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.text.method.LinkMovementMethod;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.georgcantor.wallpaperapp.R;

public class LicenseFragment extends Fragment {

    public LicenseFragment() {
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater,
                             @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.license_layout, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        Toolbar toolbar = getView().findViewById(R.id.toolbar_license);
        toolbar.setNavigationIcon(R.drawable.ic_arrow_back_white);

        ((AppCompatActivity) getActivity()).setSupportActionBar(toolbar);
        ((AppCompatActivity) getActivity()).getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        ((AppCompatActivity) getActivity()).getSupportActionBar().setTitle(getResources().getString(R.string.License));
        TextView textView = getView().findViewById(R.id.license);
        textView.setMovementMethod(LinkMovementMethod.getInstance());
        TextView textView2 = getView().findViewById(R.id.license2);
        textView2.setMovementMethod(LinkMovementMethod.getInstance());

        toolbar.setNavigationOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                getActivity().getFragmentManager().popBackStack();
            }
        });
    }
}

File: app/src/main/java/com/georgcantor/wallpaperapp/ui/fragment/LicenseFragment.kt
package com.georgcantor.wallpaperapp.ui.fragment

import android.os.Bundle
import android.support.v4.app.Fragment
import android.text.method.LinkMovementMethod
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.georgcantor.wallpaperapp.R
import kotlinx.android.synthetic.main.license_layout.*

class LicenseFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater,
                              container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.license_layout, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        license.movementMethod = LinkMovementMethod.getInstance()
        license2.movementMethod = LinkMovementMethod.getInstance()
    }
}